### ISSUE TITLE
Render code in box

### ISSUE THREAD

**Issue by carlba** [ryanpcmcquen/googleKeepCodeFormatter#2]
Would it be possible to render the code excerpts in a box which is how most markdown viewers shows it.

![screenshot 2018-10-10 14 15 11](https://user-images.githubusercontent.com/1884191/46735597-f4431d80-cc96-11e8-81ee-a560ac6f7a2a.png)

**Comment by ryanpcmcquen** [comment]
Yes, although I am quite busy at the moment. I would welcome a PR implementing this.
